bwrobot/OSGPR_GP[] localGPs
float64[] W
int32 M
int32 xdim
int32 ndim

